app/src/main/java/org/simple/clinic/bloodsugar/entry/BloodSugarEntryUiRenderer.kt

```diff
@@ -25,6 +25,7 @@ class BloodSugarEntryUiRenderer(
 
   private fun showEnterNewBloodSugarTitle(measurementType: BloodSugarMeasurementType) {
     with(ui) {
+      hideRemoveButton()
```

Comment:
Same comment as earlier about the method being here because of convenience, and moving it into the `render` function.

Reply:
Done 👍